R/read_package.R

```diff
@@ -5,6 +5,8 @@
 #' that describes the Data Package metadata and its Data Resources.
 #'
 #' @param file Path or URL to a `datapackage.json` file.
+#' @param quiet A logical. If `TRUE` no reminder about citations/rights
```

Comment:
Would simplify to:

> If `TRUE`, do not show message.

Reply:
It only silences the specific citation/rights message though, not any of the error messages from assertthat, for example:

```r
  assertthat::assert_that(
    is.character(file),
    msg = "`file` must be a path or URL to a `datapackage.json` file."
  )
```

I want to be clear because the effect is so specific. I'm ok with dropping "a logical"

Reply:
What about: 

> If `TRUE`, do not show message about access rights